autogpt_platform/backend/backend/blocks/google/gmail.py

```diff
@@ -23,6 +23,11 @@
 )
 
 
+def parse_email_recipients(recipients: list[str]) -> str:
```

Comment:
the function name doesn't align with what the function is doing, 
serialize_email_recipients?